mantidproject/mantidimaging
# Convert larger tools to windows

Converts filters, COR/tilt and reconstruction to windows.

Fixes an issue with stock selector widgets not having connections to the main window removed.

To test:
- Open each window and make sure they still load
- Run a filter
- Open and close some of the mentioned windows
- Load and delete data (doing this after one of the mentioned windows has been closed demonstrates the mentioned bug is fixed)
